### DISCUSSION TITLE
Update toolkit to no list grafana-ui and related projects in the main changelog

### DISCUSSION THREAD

**Discussion by xlson** [grafana/grafana#38155] [Dashboards, Panels & Visualizations]
**What would you like to be fixed**:

When running grafana-toolkit changelog we list changes related to the js libraries `grafana-*` twice, once in the main changelog and once in the respective sections. We should only list changes once, presumably in the right sections.

Example from the 7.2.0 release yesterday:

```
$ ./node_modules/.bin/grafana-toolkit changelog -m 195
Running in local/linked mode
⠴ Generating changelog

  /*** Grafana changelog  ***/

  ### Features / Enhancements
  * ........ (removed snippet) 
  * **grafana/ui**: Do not bundle jQuery. [#27667](https://github.com/grafana/grafana/pull/27667), [@kennytm](https://github.com/kennytm)

  ### Bug Fixes
  * ........ (removed snippet) 
  * **grafana-toolkit**: avoid path.resolve with globby in moveStaticFiles. [#27670](https://github.com/grafana/grafana/pull/27670), [@kennytm](https://github.com/kennytm)


  /*** grafana-toolkit changelog  ***/


  ### Bug Fixes
  * **grafana-toolkit**: avoid path.resolve with globby in moveStaticFiles. [#27670](https://github.com/grafana/grafana/pull/27670), [@kennytm](https://github.com/kennytm)


  /*** grafana-ui changelog  ***/

  ### Features / Enhancements
  * **grafana/ui**: Do not bundle jQuery. [#27667](https://github.com/grafana/grafana/pull/27667), [@kennytm](https://github.com/kennytm)

✔ Generating changelog
```

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
